demo/boundingbox/src/index.html

```diff
@@ -13,7 +13,7 @@
 
   <div class="boundingBox" data-service="faces" data-image-list="http://i.imgur.com/QiksDIs.jpg,https://www.nbc.com/sites/nbcunbc/files/files/styles/1080xauto/public/scet/photos/269/7924/PAA_267.JPG,http://wiinoob.walyou.netdna-cdn.com/wp-content/uploads/2009/03/mii-cupcakes.jpg"></div>
   <hr/>
-  <div class="boundingBox" data-service="face_gender" data-image-list="http://i.imgur.com/QiksDIs.jpg,https://www.nbc.com/sites/nbcunbc/files/files/styles/1080xauto/public/scet/photos/269/7924/PAA_267.JPG,http://wiinoob.walyou.netdna-cdn.com/wp-content/uploads/2009/03/mii-cupcakes.jpg"></div>
+  <div class="boundingBox" data-service="face_gender" data-image-list="http://i.imgur.com/QiksDIs.jpg,https://www.nbc.com/sites/nbcunbc/files/files/styles/1080xauto/public/scet/photos/269/7924/PAA_267.JPG,http://wiinoob.walyou.netdna-cdn.com/wp-content/uploads/2009/03/mii-cupcakes.jpg" data-confidence="0.5"></div>
```

Comment:
Just a detail, would be great to keep the parameter's name from API, `confidence_threshold`, possibly changed as `confidence-threshold` if needed.